### PR TITLE
CK-27: Conversations list UI

### PR DESCRIPTION
**Pull request by TimurM**
For this PR, in the App.js file make sure to include ConversationsListView component.

### CODE REVIEW

File: platforms/chat/.expo/packager-info.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "expoServerPort": 19000,
-  "packagerPort": 19001,
-  "packagerPid": 38677,
-  "expoServerNgrokUrl": "https://3i-pvb.patrickxrivera.chat.exp.direct",
-  "packagerNgrokUrl": "https://packager.3i-pvb.patrickxrivera.chat.exp.direct",
-  "ngrokPid": 38688
+  "expoServerPort": 19010,
```
Comment (devmeyster) [inline platforms/chat/.expo/packager-info.json]: Lets remove this file and add it to git ignore

File: platforms/chat/.expo/settings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (devmeyster) [inline platforms/chat/.expo/settings.json]: remove this file as well

File: platforms/chat/src/components/ConversationSummary.component.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  render() {
+    const { conversation } = this.props;
+    //1. replace with a flag if a new message is present.
+    //conversion from true to 1 or false to 0 is to avoid a warning messages
+    //when passing a unread="true" because unread property will be attached to the
+    //Dom Node as an attribute
+    const unreadMessage = this.state.unread ? 1 : 0;
+
+    return (
+      <TouchableOpacity onPress={this.messageRead}>
```
Comment (devmeyster) [inline platforms/chat/src/components/ConversationSummary.component.js]: I would generalize the names of the components. If making the component itself fit multiple use cases is to much, lets at least give it a generic name so the next person using this component can add extra functionality.

Comment (TimurM) [reply platforms/chat/src/components/ConversationSummary.component.js]: we'll revisit this later, once we plan out our shared components design

**Pull request merged**